packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -786,6 +787,8 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 						: "GC_Deleted_Blob_Requested",
 				category: shouldFail ? "error" : "generic",
 				isSummarizerClient: this.runtime.clientDetails.type === summarizerClientType,
+				gcEnforcementDisabled: (this.runtime as ContainerRuntime)
+					.disableGcTombstoneEnforcement,
```

Comment:
Why are we logging just the runtime level event instead of throwOnTombstoneLoad?

Reply:
throwOnTombstoneLoad is generally encapsulated in the category